nasa-gibs/worldview
# WV-2094: Tracking Data Collections

## Description

This adds a collection identification badge for layers that have been fully transitioned from GIBS to GITC. The badge will identify if the layer is currently utilizing NRT or Standard data and the version of the collection.

## How To Test
1. `git checkout tracking-data-collection-dates-update`
2. `npm ci`
3. `npm run watch`
4. [Use this link to load layers that have been transitioned to GITC.](http://localhost:3000/?v=-218.12488563727786,-136.40800990028663,148.59126442506462,149.963965466579&l=Reference_Labels_15m,Reference_Features_15m(hidden),Coastlines_15m,VIIRS_SNPP_Aerosol_Type_Deep_Blue_Best_Estimate,VIIRS_NOAA20_Cloud_Optical_Thickness,VIIRS_NOAA20_Clear_Sky_Confidence_Day,VIIRS_CrIS_SNPP_BT_Band33_Fusion_Day,BlueMarble_NextGeneration(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=false&t=2022-12-12-T17%3A52%3A53Z)
5. Verify that the following 4 layers have the collection identifier badge 1. Deep Blue Aerosol Type 2. Cloud Optical Thickness 3. Clear Sky Confidence (Day) 4. Band 33 Brightness Temperature. 
6. Increment the date day by day until you get to the current date. The layers should eventually change to NRT or display no badge at all.
